package.json

```diff
@@ -24,6 +26,7 @@
     "@slack/webhook": "^5.0.3",
     "date-fns": "^2.16.1",
     "date-fns-tz": "^1.0.12",
+    "luxon": "^3.0.3",
```

Comment:
Hmmm I didn't realise this was already using `date-fns`. I should probably try to use this instead of luxon

Reply:
fwiw I tend to prefer `luxon` myself, which is why its being used in most of our serverless functions so far

Reply:
Me too. I'll move everything to luxon in a later PR